Foundation/Operation.swift

```diff
@@ -168,6 +168,21 @@ open class Operation : NSObject {
 #endif
         _ready = true
     }
+
+    internal func _checkReadyness() -> Bool {
+        if _ready {
+            return true
+        }
+
+        #if DEPLOYMENT_ENABLE_LIBDISPATCH
```

Comment:
I don't think at this point that we have a viable setup wherein we ship Foundation without libdispatch, but this is for another patch to clean up, I believe.

Reply:
Yes, this is strange but I trying to save style :)